ontology.ttl

```diff
@@ -161,6 +166,27 @@ foaf:logo rdf:type owl:AnnotationProperty .
          rdfs:range :Workflow .
 
 
+###  http://odahub.io/ontology#resourceBindingEnvVarName
+:resourceBindingEnvVarName rdf:type owl:ObjectProperty ;
+                           rdfs:domain :S3 .
```

Comment:
Only S3? May it be applicable to the whole StorageResource?

Reply:
Resolved in the latest commit